haproxy_exporter.go

```diff
@@ -211,6 +213,10 @@ func NewExporter(uri string, selectedServerMetrics map[int]*prometheus.GaugeVec,
 			42: newBackendMetric("http_responses_total", "Total of HTTP responses.", prometheus.Labels{"code": "4xx"}),
 			43: newBackendMetric("http_responses_total", "Total of HTTP responses.", prometheus.Labels{"code": "5xx"}),
 			44: newBackendMetric("http_responses_total", "Total of HTTP responses.", prometheus.Labels{"code": "other"}),
+			58: newBackendMetric("http_queue_time", "Avg. HTTP queue time for last 1024 successful connections.", nil),
```

Comment:
We generally don't mention that a metric is a exponentially moving average in the name, and I'm not aware of any naming conventions on that front.